src/EventStore.Core/Services/Transport/Tcp/ClientTcpDispatcher.cs

```diff
@@ -8,10 +8,15 @@
 using EventStore.Core.Messages;
 using EventStore.Core.Messaging;
 using EventStore.Core.Util;
+using Microsoft.AspNetCore.SignalR;
 
 namespace EventStore.Core.Services.Transport.Tcp {
 	public class ClientTcpDispatcher : ClientWriteTcpDispatcher {
-		public ClientTcpDispatcher() {
+		public ClientTcpDispatcher(int commitTimeoutMs = Opts.CommitTimeoutMsDefault)
```

Comment:
We could choose not to have a default parameter here but to update the tests and revert the change in `Opts.cs`.